extension/parquet/column_reader.cpp

```diff
@@ -1464,6 +1464,39 @@ unique_ptr<ColumnReader> ColumnReader::CreateReader(ParquetReader &reader, const
 			break;
 		}
 		break;
+	case LogicalTypeId::TIMESTAMP_NS:
+		switch (schema_p.type) {
+		case Type::INT96:
+			return make_uniq<CallbackColumnReader<Int96, timestamp_ns_t, ImpalaTimestampToTimestampNS>>(
+			    reader, type_p, schema_p, file_idx_p, max_define, max_repeat);
+		case Type::INT64:
+			if (schema_p.__isset.logicalType && schema_p.logicalType.__isset.TIMESTAMP) {
```

Comment:
Do we have some (small) Parquet files that we can use to test this functionality? We could perhaps generate some with pyarrow?

Reply:
I do actually but when I got reading and writing working with comparing to queries against text it seemed redundant.

Reply:
Do you mean files with `Int96` timestamps?